scalafix-testkit/src/main/scala/scalafix/testkit/SemanticRuleSuite.scala

```diff
@@ -121,7 +105,24 @@ abstract class SemanticRuleSuite(
                  |$tried""".stripMargin)
           }
         }
-      assertNoDiff(obtained, expected)
+
+      val expectedLintMessages = CommentAssertion.extract(ctx.tokens)
+      val diff = AssertDiff(obtainedLintMessages, expectedLintMessages)
+
+      if (diff.isFailure) {
+        println("###########> Lint       <###########")
+        println(diff.toString)
+      }
+
+      val result = compareContents(obtained, expected)
+      if (result.nonEmpty) {
+        println("###########> Diff       <###########")
+        println(error2message(obtained, expected))
```

Comment:
Is the motivation for this change to copy-paste the terminal output? If so, we should include the filename since println is detached from the scalatest failure report.